doc/manual.rst

```diff
@@ -0,0 +1,116 @@
+.. Copyright (c) 2008-2020 OpenShot Studios, LLC
+ (http://www.openshotstudios.com). This file is part of
+ OpenShot Video Editor (http://www.openshot.org), an open-source project
+ dedicated to delivering high quality video editing and animation solutions
+ to the world.
+
+.. OpenShot Video Editor is free software: you can redistribute it and/or 
+ modify  it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+.. OpenShot Video Editor is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+.. You should have received a copy of the GNU General Public License
+ along with OpenShot Library.  If not, see <http://www.gnu.org/licenses/>.
+
+.. Edited September 27, 2020 - [USA]TechDude
+
+User Manual
+===================
+Thank you for your interest in OpenShot Video Editor, the easy to use, quick to 
+learn, and powerful video editor.  OpenShot helps you start any video project
+by importing and editing your video, audio or pictures, adding special effects
+and transitions and sharing your final product on social media sites, personal
+devices or permanent storage.
+
+.. image:: images/title-ops.png
+
+System Requirements
+-------------------
+Video editing benefits from large amounts of memory, modern multi-core CPUs, 
+and fast internal storage.  When editing video, you want the best computer 
+system that you can afford.  Here are the minimum and the recommended system 
+requirements:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Minimum System Requirements
+     - Recommended System Requirements
```

Comment:
I would recommend 16 GB still (unless that is not very common anymore). 
macOS minimum version is now 10.11 (I'm pretty sure)

Reply:
Ubuntu minimum 18.04